code/game/objects/items/storage/boxes.dm

```diff
@@ -1259,3 +1259,12 @@
 	var/obj/item/stack/sheet/cardboard/I = new(user.drop_location())
 	qdel(src)
 	user.put_in_hands(I)
+
+/obj/item/storage/box/marshmallow
+	name = "box of marshmallows"
+	desc = "A box of marshmallows."

```

Comment:
Redundant.

Reply:
yes I know, I still like this description